lib/utils/addEntries.js

```diff
@@ -2,123 +2,138 @@
 
```

Comment:
If I rename this file, GitHub doesn't seem to recognize it has moved. I'll rename the file name after review.